packages/app/src/components/logo/index.tsx

```diff
@@ -25,4 +29,27 @@ const Logo: React.FC<Props> = ({ left, right, className = '', onClick }) => {
     </svg>
   );
 };
+
+const Renewal: React.FC<Props> = ({ className = '', onClick }) => {
```

Comment:
This change could possibly make the Logo less visible on some themes, because there are multiple themes for Viron.

In my opinion, the Viron logo should have a choice of left and right colors.

Reply:
I restored the old logo.
e25d50d3e682162645d1df5f587d692e7e102bcc